client/tso_client.go

```diff
@@ -118,7 +118,9 @@ func (c *tsoClient) getOption() *option { return c.option }
 func (c *tsoClient) getServiceDiscovery() ServiceDiscovery { return c.svcDiscovery }
 
 func (c *tsoClient) setup() {
-	c.svcDiscovery.CheckMemberChanged()
+	if err := c.svcDiscovery.CheckMemberChanged(); err != nil {
+		log.Warn("[tso] failed to check member changed", errs.ZapError(err))
```

Comment:
using `log.warn` or `log.error`?

Reply:
I'm not very sure either, because this error was ignored before, so I used "warn"

Reply:
It's ok to use warn because `CheckMemberChanged` will also output some error logs internally.